modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -736,9 +746,76 @@ def map_axis_partitions(
             right=None,
             lengths=lengths,
             enumerate_partitions=enumerate_partitions,
+            apply_func_args=map_func_args,
+            apply_func_kwargs=map_func_kwargs,
             **kwargs,
         )
 
+    @classmethod
+    def map_partitions_splitting_by_column(
+        cls,
+        partitions,
+        column_splits,
+        map_func,
+        map_func_args=None,
+        map_func_kwargs=None,
+    ):
+        """
+        Combine several blocks by column into one virtual partition and apply “map_funk” to them.
+
+        Parameters
+        ----------
+        partitions : NumPy 2D array
+            Partitions of Modin Frame.
+        column_splits : int
+            The number of splits by column.
+        map_func : callable
+            Function to apply.
+        map_func_args : iterable, optional
+            Positional arguments for the 'map_func'.
+        map_func_kwargs : dict, optional
+            Keyword arguments for the 'map_func'.
+
+        Returns
+        -------
+        NumPy array
+            An array of new partitions for Modin Frame.
+        """
+        new_partitions = np.array(
+            [
+                cls.column_partitions(
+                    partitions[i : i + column_splits],
+                    # full_axis=False,
```

Comment:
As I see, `full_axis` is used to rebalance the dataframe when some blocks are combined into one virtual partition and it should to be saved further.
In this case, we don't need to save the new partitionning, so we don't need to use this argument.
Note that the temporary partitioning includes several column partitions per column, but after applying map_func it is reverted back.

I left this line just for review. If it is accepted, I will remove it.

Reply:
Let's remove it.